client/state/themes/test/reducer.js

```diff
@@ -402,4 +408,220 @@ describe( 'reducer', () => {
 			expect( state ).to.eql( {} );
 		} );
 	} );
+
+	describe( '#activeThemes()', () => {
+		it( 'should default to an empty object', () => {
+			const state = activeThemes( undefined, {} );
+
+			expect( state ).to.eql( {} );
+		} );
+
+		it( 'should track active theme request success', () => {
+			const state = activeThemes( deepFreeze( {} ), {
+				type: ACTIVE_THEME_REQUEST_SUCCESS,
+				siteId: 2211667,
+				themeId: 'rebalance',
+				themeName: 'Rebalance',
+				themeCost: {
+					currency: 'USD',
+					number: 0,
+					display: ''
+				}
+			} );
+
+			expect( state ).to.have.keys( [ '2211667' ] );
```

Comment:
Just a testing helper that allows you to know faster exactly what has happened. Of course next line would also allow you to know that.  Sinon expect fail messages are more verbose this way as they target differen issues. First time it is presence of key and second time is the correctness of whole struct.